docs/release_notes/v-1.5.md

```diff
@@ -0,0 +1,67 @@
+# Airbyte 1.5.0
```

Comment:
Can we please add:
1. (Enterprise) Connector Resource Allocation
2. Bug fix for  https://github.com/airbytehq/airbyte/issues/46097 (you can talk with Jon Tan about this)